lib/builder-browserify.js

```diff
@@ -71,7 +71,9 @@ module.exports = function (files, config) {
   }
 
   bundle.close = function () {
-    watchifyBundler.close()
+    if (typeof watchifyBundler.close === 'function') {
```

Comment:
Maybe do `if (config.watchify)` here too, that's more obvious than `if (typeof watchifyBundler.close === 'function')`.